content/influxdb/clustered/admin/custom-partitions/view-partitions.md

```diff
@@ -0,0 +1,199 @@
+---
+title: View partition information
+description: >
+  Query partition information from InfluxDB v3 system tables to view partition 
+  templates and verify partitions are working as intended.
+menu:
+  influxdb_clustered:
+    name: View partitions
+    parent: Manage data partitioning
+weight: 202
+list_code_example: |
+  ```sql
+  SELECT * FROM system.partitions WHERE table_name = 'example-table'
+  ```
+related:
+  - /influxdb/clustered/admin/query-system-data/
+---
+
+{{< product-name >}} stores partition information in InfluxDB v3 system tables.
+Query partition information to view partition templates and verify partitions
+are working as intended.
+
+- [Query partition information from system tables](#query-partition-information-from-system-tables)
+- [Partition-related queries](#partition-related-queries)
+
+{{% warn %}}
+#### Querying system tables may impact overall cluster performance
+
+Partition information is stored in InfluxDB v3 system tables.
+Querying system tables may impact the overall write and query performance of
+your {{< product-name omit=" Clustered" >}} cluster.
+
+<!--------------- UPDATE THE DATE BELOW AS EXAMPLES ARE UPDATED --------------->
+
+#### System tables are subject to change
+
+System tables are not part of InfluxDB's stable API and may change with new releases.
+The provided schema information and query examples are valid as of **August 22, 2024**.
+If you detect a schema change or a non-functioning query example, please
+[submit an issue](https://github.com/influxdata/docs-v2/issues/new/choose).
+
+<!--------------- UPDATE THE DATE ABOVE AS EXAMPLES ARE UPDATED --------------->
+
+{{% /warn %}}
+
+## Query partition information from system tables
+
+Use the [`influxctl query` command](/influxdb/clustered/reference/cli/influxctl/query/)
+and SQL to query partition-related information from InfluxDB system tables.
+ Provide the following:
+
+- **Enable system tables** with the `--enable-system-tables` command flag.
+- **Database token**: A [database token](/influxdb/clustered/admin/tokens/#database-tokens)
+  with read permissions on the specified database. Uses the `token` setting from
+  the [`influxctl` connection profile](/influxdb/clustered/reference/cli/influxctl/#configure-connection-profiles)
+  or the `--token` command flag.
+- **Database name**: The name of the database to query information about.
+  Uses the `database` setting from the
+  [`influxctl` connection profile](/influxdb/clustered/reference/cli/influxctl/#configure-connection-profiles)
+  or the `--database` command flag.
+- **SQL query**: The SQL query to execute.
+  Pass the query in one of the following ways:
+
+  - a string on the command line
+  - a path to a file that contains the query
+  - a single dash (`-`) to read the query from stdin
+
+{{% code-placeholders "DATABASE_(TOKEN|NAME)|SQL_QUERY" %}}
+
+<!--pytest.mark.skip-->
```

Comment:
This test should run successfully against Cloud Dedicated without any additional setup.